rules/klayout/drc/run_drc.py

```diff
@@ -188,37 +186,38 @@ def main():
             # Running DRC using klayout
             if (arguments["--antenna_only"]) and not (arguments["--density_only"]):
                 logging.info(f"Running Global Foundries 180nm MCU antenna checks on design {name_clean} on cell {topcell_name}:")
-                os.system(f"klayout -b -r $PDK_ROOT/$PDK/gf180mcu_antenna.drc -rd input={path} -rd report={name_clean}_antenna_gf{arguments['--gf180mcu']}_gf{arguments['--gf180mcu']}.lyrdb -rd thr={thrCount} {switches}")
+                os.system(f"klayout -b -r {curr_path}/gf180mcu_antenna.drc -rd input={path} -rd report={name_clean}_antenna_gf{arguments['--gf180mcu']}_gf{arguments['--gf180mcu']}.lyrdb -rd thr={thrCount} {switches}")
```

Comment:
`subprocess` here and below?

Reply:
@proppy will do.

Reply:
@proppy This entire script will be rewriten. I got the idea of your requests.